client/selfrepair/tests/test_self_repair_runner.js

```diff
@@ -17,10 +17,14 @@ const UUID_ISH_REGEX = /^[a-f0-9-]{36}$/;
 
 
 describe('Self-Repair Runner', () => {
-  afterEach(() => {
+  beforeEach(() => {
```

Comment:
I mis-remembered [this conversation](https://github.com/mozilla/normandy/pull/265#discussion_r79666402) with @brittanystoroz. I thought we had agreed that `beforeEach` was the way to do it, and so I changed it. But I was wrong, so I'll change it back.